python/aida_interchange/aifutils.py

```diff
@@ -332,7 +332,7 @@ def make_cluster_with_prototype(g, cluster_uri, prototype, system, handle=None):
     """
     cluster = _make_aif_resource(g, cluster_uri, AIDA_ANNOTATION.SameAsCluster, system)
     g.add((cluster, AIDA_ANNOTATION.prototype, prototype))
-    if handle == None:
+    if handle != None:
```

Comment:
The preferred Python syntax is "is not None"

Reply:
@jakevb I believe you can explicitly state "is not None" like @bdpham suggested: `if handle is not None:`

Reply:
Yep, I changed that in the most recent push.